public/js/controllers/main.js

```diff
@@ -129,7 +129,21 @@
 
       // essentially checks for IE/Edge
       if (window.hasOwnProperty('Notification')) {
-        Notification.requestPermission();
+        Notification.requestPermission(function (permission) {
+          // there is no point in displaying the icon if Web Notifications are not even supported
+          var $notificationIcon = $('#notification-icon');
+
+          // Set separate flag since the browser cannot reprompt
+          // or force the user for notifications once set.
+          // Separate flag is for easier toggling on the user.
+          if (permission === 'granted') {
+            sessionStorage.setItem('sfm-notifications', true);
+            $notificationIcon.addClass('fa-bell');
```

Comment:
Angular is great for avoiding DOM manipulations like this. Just create a scoped variable (c.allowNotifications or something) and then in the HTML you can display different things depending on the value of that variable. See my #player-overlay stuff for an example.